core/src/main/java/com/google/errorprone/bugpatterns/InconsistentCapitalization.java

```diff
@@ -65,7 +64,8 @@ public Description matchClass(ClassTree tree, VisitorState state) {
     ImmutableMap<String, Symbol> fieldNamesMap =
         fields
             .stream()
-            .collect(toImmutableMap(symbol -> symbol.toString().toLowerCase(), identity()));
+            .collect(
+                toImmutableMap(symbol -> symbol.toString().toLowerCase(), x -> x, (x, y) -> x));
```

Comment:
@cushon I believe that `String#toLowerCase` subtly changes behaviour depending on the default locale. Have you considered `Ascii.toLowerCase(symbol.toString())` or `symbol.toString().toLowerCase(Locale.ENGLISH)`?

Reply:
(The docs for [`String#toLowerCase(Locale)`](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#toLowerCase-java.util.Locale-) seems to recommend `toLowerCase(Locale.ROOT)` for non-locale-sensitive operations instead of say `toLowerCase(Locale.ENGLISH)`. I don't know which option of the two makes more sense here...)